src/main/java/Awex.java

```diff
@@ -0,0 +1,130 @@
+import java.util.*;
+
+public class Awex {
+    public static void message() {
+        System.out.println("Input type must be one of:");
+        System.out.println("  1. list");
+        System.out.println("  2. mark <task number>");
+        System.out.println("  3. unmark <task number>");
+        System.out.println("  4. todo <task>");
+        System.out.println("  5. deadline <task> /by <deadline>");
+        System.out.println("  6. event <task> /from <start> /to <end>");
+        System.out.println("Type 'bye' to exit.");
+    }
+
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm AWEX!\nWhat can I do for you?");
+        LinkedList<Task> list = new LinkedList<>();
+        Scanner sc = new Scanner(System.in);
+        String next = sc.nextLine();
+        String[] arr = next.split(" ");
+        while (!next.equals("bye")) {
```

Comment:
Good use of if-else statements to check and parse inputs! However, perhaps you could abstract out this method and also do consider using switch-case statements to improve performance.